src/components/Filters/FilterFile/UploadFile/DocumentsFieldArray.test.js

```diff
@@ -0,0 +1,65 @@
+import { MemoryRouter } from 'react-router-dom';
+import { Form } from 'react-final-form';
+import arrayMutators from 'final-form-arrays';
+
+import { render, screen } from '@folio/jest-config-stripes/testing-library/react';
+import userEvent from '@folio/jest-config-stripes/testing-library/user-event';
+import { StripesContext, useStripes } from '@folio/stripes/core';
+
+import DocumentsFieldArray from './DocumentsFieldArray';
+import withIntlConfiguration from '../../../../../test/jest/helpers/withIntlConfiguration';
+
+const selectedContact = jest.fn();
+
+const renderDocumentsFieldArray = (stripes) => {
+  return render(withIntlConfiguration(
+    <StripesContext.Provider value={stripes}>
+      <MemoryRouter>
+        <Form
+          onSubmit={jest.fn}
+          mutators={{
+            selectedContact,
+            ...arrayMutators,
+          }}
+          render={() => withIntlConfiguration(
```

Comment:
`withIntlConfiguration()` is used twice.

Reply:
removed one withIntlConfiguration